sdk/servicebus/azure_messaging_servicebus/tests/common/mod.rs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All Rights reserved
+// Licensed under the MIT license.
+
+use std::{env, error::Error};
+
+#[allow(dead_code)]
+static INIT_LOGGING: std::sync::Once = std::sync::Once::new();
+
+#[allow(dead_code)]
+pub fn setup() {
```

Comment:
If you use `#[recorded::test(live)]`, you get tracing for free, in addition, you can use `recording.credential()` and avoid having to configure a test credential - that way it works in both pipelines and locally.